devops/pipeline/e2etest.yaml

```diff
@@ -115,14 +118,20 @@ stages:
         matrix:
           # Find different VM variants on the Azure Marketplace
           # See https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage
-          # debian-9:
-          #   distroName: debian-9
-          #   image_publisher: credativ
-          #   image_offer: Debian
-          #   image_sku: 9-backports
-          #   image_version: 9.20210721.0
-          #   packagePattern: '**/*stretch_x86_64.deb'
-          #   device_id: debian9
+          # !!! ADD NEW DISTROS HERE !!!
```

Comment:
> # !!! ADD NEW DISTROS HERE !!! [](http://example.com/codeflow?start=9&length=31)

Same as earlier. Remove the many ! and change to lowercase

Reply:
im trying to make it obvious that its a placeholder, open to suggestions